### PR TITLE
  Fix windows python3 appveyor error - rebase

### PR DESCRIPTION
**Pull request by mpharrigan**
Rebase of #913

### CODE REVIEW

**Comment by mpharrigan** [comment]
Wait, let me rebase on #968 (whoops)

File: mdtraj/tests/test_trajectory.py [hunk 1]
```diff
@@ -502,8 +502,10 @@ def test():
 
 def test_iterload_skip():
     files = ['frame0.nc', 'frame0.h5', 'frame0.xtc', 'frame0.trr',
-             'frame0.dcd', 'frame0.binpos', 'legacy_msmbuilder_trj0.lh5',
+             'frame0.dcd', 'frame0.binpos',
              'frame0.xyz', 'frame0.lammpstrj']
+    if not (on_win and on_py3):
```
Comment (mpharrigan) [inline mdtraj/tests/test_trajectory.py]: where are `on_win` and `on_py3` supposed to be defined?

Comment (rmcgibbo) [inline mdtraj/tests/test_trajectory.py]: should be at the top of the file.

Comment (rmcgibbo) [inline mdtraj/tests/test_trajectory.py]: i'm not sure you picked up everything from that PR https://github.com/mdtraj/mdtraj/pull/913/files

**Comment by rmcgibbo** [comment]
What exactly are you trying to do here? This is really  a small subset of the changes in #913.

**Comment by mpharrigan** [comment]
Oh. I looked at the diff of #913 last week and it was a mess with a bunch of changes unrelated to windows / appveyor that looked like it was trying to revert a lot of the changes that were happening while the branch was inactive.

It looks like you updated it and now it looks sensible

**Comment by rmcgibbo** [comment]
I'll merge in the latest master into #913.

**Pull request closed without merging**